source/common/tracing/http_tracer_impl.cc

```diff
@@ -206,6 +206,11 @@ void HttpTracerUtility::finalizeDownstreamSpan(Span& span,
       addGrpcRequestTags(span, *request_headers);
     }
   }
+  span.setTag(Tracing::Tags::get().RequestSize, std::to_string(stream_info.bytesReceived()));
```

Comment:
Could we move these `RequestSize` and `ResponseSize` inside `setCommonTags`?

Reply:
(fyi: the only reason the diff shows the addition of `RequestSize` and `ResponseSize` is because I moved them up above the `CustomTagContext` loop; they were already in this function)

happy to move it, but worried it might be intentionally left out of `setCommonTags`? `setCommonTags` is called in both:
- `finalizeDownstreamSpan()`
- `finalizeUpstreamSpan()`

the `RequestSize`/`ResponseSize` are only set as tags in `finalizeDownstreamSpan`, so adding it to common-tags would put it in `finalizeUpstreamSpan` as well. Does this make sense for that method? Would those values be set?

Reply:
here's where the `bytesSent()` bit was added: https://github.com/envoyproxy/envoy/pull/8289/files#diff-36ebe2d9da39e88ceeccb197a9ff0b8a1dc97a5f72c8597c40d27f61a9ead00fR176

I guess I'd be concerned about a comment like this: https://github.com/envoyproxy/envoy/pull/8289#discussion_r326426387, where certain fields are only set up for Downstream, but not Upstream spans, for example

Reply:
ah I think its fine, followed the breadcrumbs and I don't think we need to worry about that

Reply:
@matthewgrossman The change to move `RequestSize` and `ResponseSize` might be out of range of this PR. For now It's OK not to reflect this change. Sorry for the noise ;)

Reply:
all good, was optimistic it wouldn't require too much but there's maybe slightly more there than first expected. Or just requires a simple change ¯\_(ツ)_/¯


removed the code and tests that were validating the new location